index.html

```diff
@@ -31,10 +31,11 @@
                 <p class="connect-link">LET’S CONNECT</p>
             </div>
                 <ul class="connect-icon">
-                    <a href="#" class="fa fa-twitter"></a>
-                    <a href="#" class="fa fa-linkedin"></a>
-                    <a href="#" class="fa fa-angellist"></i></a>
-                    <a href="#" class="fa fa-medium"></i></a>
+                    <li><img src="./image/Linkedinicon.png" alt="" /></li>
+                    <li><img src="./image/github.png" alt="" /></li>
+                    <li><img src="./image/twitter.png" alt="" width="20" height="20" /></li>
+                    <li><img src="./image/M.png" alt="" width="20" height="20" /></li>
+                    <li><img src="./image/fingers.png" alt="" width="20" height="20" /></li>
                 </ul>        

```

Comment:
- [ ] Please kindly fix this as pointed out by the previous reviewer, the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has an `a` link (a tag)
Example bellow :point_down: :point_down: 
```
<ul>
<li><a href="#"><img src="./image/Linkedinicon.png" alt="" width="20" height="20" /></a></li>
 ...
</ul>
```